common/buildcraft/energy/generation/OilGenerator.java

```diff
@@ -110,13 +114,13 @@ public void generate(Random random, int chunkX, int chunkZ, World world, IChunkG
         }
         final GenType type;
 
-        if (rand.nextDouble() <= CHANCE_LARGE * bonus) {
+        if (rand.nextDouble() <= BCEnergyConfig.largeOilGenProb * bonus) {
             // 0.04%
             type = GenType.LARGE;
-        } else if (rand.nextDouble() <= CHANCE_MEDIUM * bonus) {
+        } else if (rand.nextDouble() <= BCEnergyConfig.mediumOilGenProb * bonus) {
             // 0.1%
             type = GenType.MEDIUM;
-        } else if (oilBiome && rand.nextDouble() <= CHANCE_LAKE * bonus) {
+        } else if (oilBiome && rand.nextDouble() <= BCEnergyConfig.smallOilGenProb * bonus) {
```

Comment:
Maybe get rid of the constants? They won't be used anymore, unless you want to refer to them statically from the config class.